packages/js/src/novu.ts

```diff
@@ -32,7 +26,6 @@ export class Novu implements Pick<NovuEventEmitter, 'on'> {
   constructor(options: NovuOptions) {
     this.#inboxService = new InboxService({
       backendUrl: options.backendUrl ?? PRODUCTION_BACKEND_URL,
-      userAgent: options.__userAgent ?? userAgent,
```

Comment:
This is needed for useragent analytics from @novu/react and other packages

Reply:
Good catch. I will add it back.